### ISSUE TITLE
Trigger email after applicants complete registration to alert about Job picker site

### ISSUE THREAD

**Issue by arouault** [MAPC/youth-match-v2#52]
Send an email to youth who have completed an application.

**Comment by mzagaja** [comment]
Waiting on Deron to provide e-mail text and information on who it is from.

**Comment by mzagaja** [comment]
Closing as duplicate of #37.
